README.md

```diff
@@ -8,19 +8,25 @@
 Lace is a C framework for fine-grained fork-join parallelism on multi-core computers.
 
 ```c
-TASK_1(int, fibonacci, int, n) {
+TASK_1(int, fibonacci, int, n)   // macro to create Lace functions (can be in header file)
+
+int fibonacci(LaceWorker* lw, int n) {
     if(n < 2) return n;
-    SPAWN(fibonacci, n-1);
-    int a = CALL(fibonacci, n-2);
-    int b = SYNC(fibonacci);
-    return a+b;
+    fibonacci_SPAWN(lw, n-1);    // SPAWN a task (fork)
+    int a = fibonacci(lw, n-2);  // run another task in parallel
+    int b = fibonacci_SYNC(lw);  // SYNC the spawned task (join)
+    return a + b;
 }
 
-int main(int argc, char **argv)
+int main(int argc, char** argv)
 {
-    int n_workers = 4;
-    lace_start(n_workers, 0);
-    int result = RUN(fibonacci, 42);
+    int n_workers = 4;           // create 4 workers
+                                 // use 0 to automatically use all available cores
+    int dqsize = 0;              // use default task deque size
+    int stacksize = 0;           // use default program stack size
+
+    lace_start(n_workers, dqsize, stacksize);
+    int result = fibonacci_RUN(42);
     printf("fibonacci(42) = %d\n", result);
     lace_stop();
 }
```

Comment:
Maybe add a "`For more examples, see [DOCS.md](./DOCS.md) and the contents of the [benchmarks](./benchmarks/) folder.`" ? It is also at the bottom, but one may as well just mention `DOCS.md` twice.